packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Deployment/Deployment.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+
+import { useSignals } from 'app/store';
+
+import { Description } from '../../elements';
+
+import { Netlify } from './Netlify';
+import { Zeit } from './Zeit';
+
+export const Deployment = () => {
+  const {
+    deployment: { getDeploys },
+  } = useSignals();
+
+  useEffect(() => getDeploys(), []); // eslint-disable-line
```

Comment:
@Saeris It's a best practice to add the `getDeploys ` function to the `useEffect` dependencies even though you only want the effect to run on mount

```suggestion
  useEffect(() => getDeploys(), [getDeploys]);
```

Reply:
Agree, we can add it as commit!

Reply:
(I don't have access to accept the suggestion)

Reply:
Also can't commit the suggested change. GitHub is telling me I don't have permission to push.

But yeah this change is fine. I haven't included signals as dependencies because they should never change.